release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -849,6 +849,31 @@ Helm-based Operator enhancements include:
 {product-title} now properly supports the `terminationGracePeriodSeconds`
 parameter with the CRI-O container runtime.
 
+[discrete]
+[id="ocp-4-5-readyz-config"]
+==== /readyz configuration for API server health probe
+
+All {product-title} 4.5 clusters must be configured to use the `/readyz`
+endpoint for API server health checking to remain supported. Any clusters
+installed on versions prior to {product-title} 4.5 must be reconfigured to use
+`/readyz`.
+
+Clusters using user-provisioned infrastructure without `/readyz` configured can
+suffer from API outages when the API server restarts. This can be caused by
+events such as configuration changes, certificate updates, or master node
+reboots.
+
+The load balancer should be configured to take a maximum of 30 seconds from the
+time the API server turns off the `/readyz` endpoint to the removal of the API
+server instance from the pool. Within the time frame after `/readyz` returns an
+error or becomes healthy, the endpoint must have been removed or added. Probing
+every 5 or 10 seconds, with two successful requests to become healthy and three
```

Comment:
This could just be my lack of understanding, but would three _successful_ requests indicate an unhealthy status, or three _unsuccessful_ requests?

Reply:
As I understand the recommendation, the probe should make two consecutive successful requests for the kube-apiserver to be considered healthy. It should make three consecutive failed requests for the kube-apiserver to be considered unhealthy. This aligns with the `successThreshold` and `failureThreshold` params defined [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes). I'll tweak this language just a bit for clarity.

Reply:
This still isn't super clear.

Reply:
Sorry, I hadn't updated that part yet. Doing it now :+1: